src/main/scala/mesosphere/marathon/core/matcher/manager/OfferMatcherManagerModule.scala

```diff
@@ -0,0 +1,30 @@
+package mesosphere.marathon.core.matcher.manager
+
+import akka.actor.ActorRef
+import mesosphere.marathon.core.base.Clock
+import mesosphere.marathon.core.leadership.LeadershipModule
+import mesosphere.marathon.core.matcher.base.OfferMatcher
+import mesosphere.marathon.core.matcher.manager.impl.{ OfferMatcherManagerDelegate, OfferMatcherManagerActor }
+import mesosphere.marathon.core.matcher.base.util.ActorOfferMatcher
+import mesosphere.marathon.metrics.Metrics
+import rx.lang.scala.subjects.PublishSubject
+import rx.lang.scala.{ Observable, Subject }
+
+import scala.util.Random
+
+class OfferMatcherManagerModule(
```

Comment:
Please add some documentation for this module as well the provided objects/functionality.